cpan/ExtUtils-PL2Bat/t/make_executable.t

```diff
@@ -31,3 +32,29 @@ push @files, grep { -f } map { $filename.$_ } split / $Config{path_sep} /x, $ENV
 is scalar(@files), 1, "Executable file exists";
 
 unlink $filename, @files;
+
+# the test needs CWD in PATH to check the created .bat files, but under win2k
+# PATH must not be too long. so to keep any win2k smokers happy, we construct
+# a new PATH that contains the dirs which hold cmd.exe, perl.exe, and CWD
+
+sub construct_test_PATH {
+	my $perl_path = $^X;
+	my $cmd_path = $ENV{ComSpec} ||  `where cmd`; # where doesn't seem to work on all windows versions
+	$_ =~ s/[\\\/][^\\\/]+$// for $perl_path, $cmd_path; # strip executable names
+
+	my @path_fallbacks = grep /\Q$ENV{SystemRoot}\E|system32|winnt|windows/i, split $Config{path_sep}, $ENV{PATH};
+
+	my $path_with_cwd = join $Config{path_sep}, @path_fallbacks, $cmd_path, $perl_path, cwd();
+
+	my ($perl) = ( $^X =~ /[\\\/]([^\\]+)$/ ); # in case the perl executable name differs
+	note "using perl executable name: $perl";
+
+	local $ENV{PATH} = $path_with_cwd;
+	my $test_out = `$perl -e 1 2>&1`;
```

Comment:
maybe check `$?` in addition to the output?

Reply:
Yeah, i can add that.